src/analyze/accumulatedtracedata.h

```diff
@@ -172,6 +173,17 @@ struct AccumulatedTraceData
     std::vector<IpIndex> opNewIpIndices;
 
     std::vector<AllocationInfo> allocationInfos;
+
+    struct ParsingState {
+        int64_t fileSize_b = 0;
+        int64_t compressedByte = 0;
+        int64_t uncompressedByte = 0;
+        int64_t timestamp_ms = 0;
```

Comment:
just timeStamp

Reply:
Will change if you feel strongly about it, but I've been bitten previously so many times in my career by doing math with implicit & mismatched units that I'd resolved to always include units where possible either via type safety or if not then by var name.

Reply:
Type safety would be fine - but we only have that for time via `std::chrono` so far...

Anyhow, the rest of the code base operates on ms timestamps anyways (cf. `Util::formatTime`) and bytes (`Util::formatBytes`) and I don't have that unit stored anywhere else in the variable name. So I'd prefer if this stays uniform.

If you'd like, I'd be OK to change the timestamps everywhere to chrono in a follow-up patch, but that shouldn't be done within this one here as it's not directly related.

Reply:
done.